src/ARMS/chassis.cpp

```diff
@@ -453,22 +457,20 @@ void init(std::initializer_list<okapi::Motor> leftMotors,
 		                                            std::get<3>(encoderPorts));
 	}
 
-	chassis::encodersReversed = encoderReversed;
```

Comment:
The encoderReversed parameter needs removed from the function prototype in both in chassis.cpp and chassis.h since it is no longer being used.